tests/UnifiedSpecTests/Util.php

```diff
@@ -26,13 +34,113 @@
+            'deleteMany' => ['filter', 'session', 'collation', 'hint'],
+            'deleteOne' => ['filter', 'session', 'collation', 'hint'],
+            'findOneAndDelete' => ['filter', 'session', 'typeMap', 'projection', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint', 'maxTimeMS', 'new', 'sort', 'update', 'upsert'],
+            'distinct' => ['fieldName', 'filter', 'session', 'typeMap', 'collation', 'maxTimeMS'],
+            'drop' => ['session', 'typeMap'],
+            'find' => ['filter', 'session', 'typeMap', 'allowDiskUse', 'allowPartialResults', 'batchSize', 'collation', 'comment', 'cursorType', 'hint', 'limit', 'max', 'maxAwaitTimeMS', 'maxScan', 'maxTimeMS', 'min', 'modifiers', 'noCursorTimeout', 'oplogReplay', 'projection', 'returnKey', 'showRecordId', 'skip', 'snapshot', 'sort'],
+            'findOne' => ['filter', 'session', 'typeMap', 'allowDiskUse', 'allowPartialResults', 'batchSize', 'collation', 'comment', 'cursorType', 'hint', 'max', 'maxAwaitTimeMS', 'maxScan', 'maxTimeMS', 'min', 'modifiers', 'noCursorTimeout', 'oplogReplay', 'projection', 'returnKey', 'showRecordId', 'skip', 'snapshot', 'sort'],
+            'findOneAndReplace' => ['returnDocument', 'filter', 'replacement', 'session', 'typeMap', 'projection', 'returnDocument', 'upsert', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint', 'maxTimeMS', 'new', 'remove', 'sort'],
+            'replaceOne' => ['filter', 'replacement', 'session', 'upsert', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint'],
+            'findOneAndUpdate' => ['returnDocument', 'filter', 'update', 'session', 'typeMap', 'upsert', 'projection', 'remove', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint', 'maxTimeMS', 'sort'],
+            'updateMany' => ['filter', 'update', 'session', 'upsert', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint'],
+            'updateOne' => ['filter', 'update', 'session', 'upsert', 'arrayFilters', 'bypassDocumentValidation', 'collation', 'hint'],
+            'insertMany' => ['options', 'documents', 'session', 'ordered', 'bypassDocumentValidation'],
+            'insertOne' => ['document', 'session', 'bypassDocumentValidation'],
+            'listIndexes' => ['session', 'maxTimeMS'],
+            'mapReduce' => ['map', 'reduce', 'out', 'session', 'typeMap', 'bypassDocumentValidation', 'collation', 'finalize', 'jsMode', 'limit', 'maxTimeMS', 'query', 'scope', 'sort', 'verbose'],
+        ],
+        ChangeStream::class => [
+            'iterateUntilDocumentOrError' => [],
+        ],
+        Cursor::class => [
+            'close' => [],
+            'iterateUntilDocumentOrError' => [],
+        ],
+        Session::class => [
+            'abortTransaction' => [],
+            'commitTransaction' => [],
+            'endSession' => [],
+            'startTransaction' => [],
+            'withTransaction' => ['callback', 'readConcern', 'writeConcern'],
```

Comment:
```suggestion
            'withTransaction' => ['callback', 'maxCommitTimeMS', 'readConcern', 'readPreference', 'writeConcern'],
```

In the test runner's operation class, all arguments other than `callback` end up being passed as the `$transactionOptions` argument to `with_transaction()` (see: [here](https://github.com/mongodb/mongo-php-library/blob/1.11.0/tests/UnifiedSpecTests/Operation.php#L633)). In turn, [`with_transaction()`](https://github.com/mongodb/mongo-php-library/blob/1.11.0/src/functions.php#L429) passes those options as-is on to the [`WithTransaction`](https://github.com/mongodb/mongo-php-library/blob/1.11.0/src/Operation/WithTransaction.php) operation, where they are eventually passed on to `startSession()` in [`WithTransaction::execute()`](https://github.com/mongodb/mongo-php-library/blob/1.11.0/src/Operation/WithTransaction.php#L64).

Therefore, this operation should support the same arguments as `startTransaction` with the addition of `callback`.